src/tool.js

```diff
@@ -2,7 +2,7 @@ exports.genURL = (urlLength) => {
   var urlString = "";
   const sourceLetters = "ABCDEFGHJKMNPQRSTUVWXYZabcdefghjkmnpqrstuvwxyz23456789";
 
-  for(var genLoopIndex = 0; genLoopIndex < 50; genLoopIndex++){
```

Comment:
error received:
`1) URL Gen Testing
       Was the proper length observed:

      AssertionError: expected 50 to equal 150
      + expected - actual

      -50
      +150
`